Umbraco-StarterKit/Manage-users/index.md

```diff
@@ -4,4 +4,27 @@ versionFrom: 8.0.0
 
 # Manage users
 
-In this section you can find all the information and details you need in order to work with and manage both users and members on your website.
+In this section, you can find all the information and details you need in order to work with and manage both users and members on your website.
+
+## What is a User
+
+A user is someone who has access to the Umbraco backoffice and is not to be confused with Members. When Umbraco has been installed a user will automatically be generated with the login (email) and password entered during installation. Users can be created, edited, and managed in the User section.
+Managing your users.
+You can manage your users so they can’t publish or edit only read, or you can disable someone from your project if they should not have access anymore.
+
+Please note that for now a user cannot be deleted, they can only be disabled.
+
+You will find that your project has a user named themeowner, it is there to make your project work.
```

Comment:
Perhaps we can put this in a note, tip or warning box, to highlight it a bit more?

And I think we also need to explain it in more detail, why that user needs to be there 🤔